homeassistant/components/alexa/smart_home.py

```diff
@@ -1416,6 +1415,39 @@ def temperature_from_object(hass, temp_obj, interval=False):
     return convert_temperature(temp, from_unit, to_unit, interval)
 
 
+def _convert_ha_mode(ha_mode):
+    mode = next(
```

Comment:
This code is not readable, please rewrite with a simple for loop.

Reply:
👍 No problem.  will do.  
fyi, I just did exactly what the old code did